STYLE.md

```diff
@@ -43,6 +43,9 @@
   * `using BarSharedPtr = std::shared_ptr<Bar>;`
   * `using BlahConstSharedPtr = std::shared_ptr<const Blah>;`
   * Regular pointers (e.g. `int* foo`) should not be type aliased.
+* `absl::optional<std::reference_wrapper<T>> is type aliased:
+  * `using FooOptRef = absl::optional<std::reference_wrapper<T>>;`
+  * `using FooOptConstRef = absl::optional<std::reference_wrapper<T>>;`
```

Comment:
We should probably expand the language on line 41 beyond "smart pointers".

Reply:
@derekargueta not sure what you mean? I see these two as different bullet points.

Reply:
oh sorry, missed the un-indentation on 46 to make it a new section 👍